core/arch/arm/plat-rockchip/platform_rk3588.c

```diff
@@ -49,37 +50,9 @@
 #define TRNG_POLL_PERIOD_US	0
 #define TRNG_POLL_TIMEOUT_US	1000
 
-#define OTP_S_AUTO_CTRL	0x0004
-#define OTP_S_AUTO_EN	0x0008
-#define OTP_S_PROG_DATA	0x0010
-#define OTP_S_DOUT	0x0020
-#define OTP_S_INT_ST	0x0084
-
-#define ADDR_SHIFT	16
-#define BURST_SHIFT	8
-#define CMD_READ	0
-#define CMD_WRITE	2
-#define EN_ENABLE	1
-#define EN_DISABLE	0
-
-#define MAX_INDEX	0x300
-#define BURST_SIZE	8
-#define OTP_WORD	1
-
-#define OTP_S_ERROR_BIT		BIT32(4)
-#define OTP_S_WR_DONE_BIT	BIT32(3)
-#define OTP_S_VERIFY_BIT	BIT32(2)
-#define OTP_S_RD_DONE_BIT	BIT32(1)
-
-#define OTP_POLL_PERIOD_US	0
-#define OTP_POLL_TIMEOUT_US	1000
-
-#define HW_UNIQUE_KEY_INDEX	0x104
```

Comment:
`HW_UNIQUE_KEY_INDEX` is only used by `tee_otp_get_hw_unique_key()` that remains in *core/arch/arm/plat-rockchip/platform_rk3588.c*. I think the macro definition can remain in this source file.

Reply:
`HW_UNIQUE_KEY_INDEX` describes the layout of the data in the OTP fuses. There will be other constants for other data, for example the secure boot status or the public key hash. The `HW_UNIQUE_KEY_INDEX` and other indexes are API for how the OTP driver shall be used and, thus, belongs into the header.

Reply:
Move the macro definition to *platform_config.h* if it's platform flavor specific?
But okay if you really think it deserves to be defined in that driver header file.

That said, being moved outside a specific source file, I think it should also be renamed with `ROCKCHIP_OPT_` or like prefix.